common/parse.go

```diff
@@ -125,3 +125,24 @@ func getRecords(optional bool, jsonPath string, nestedPath ...string) RecordsFun
 		return jsonquery.Convertor.ArrayToMap(arr)
 	}
 }
+
+// ParseLinkHeader extracts the next page URL from the Link Header response.
+func ParseNexPageLinkHeader(linkHeader string) string {
+	var url string
+
+	if linkHeader == "" {
+		return "" // this indicates we're done.
+	}
+
+	links := strings.Split(linkHeader, ",")
```

Comment:
Is there a better way to parse this with the url package? This seems a little brittle